src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -286,11 +283,9 @@ protected ResponseEntity inviteUserToOrganisation(NewUserCreationRequest newUser
         Organisation existingOrganisation = organisationService.getOrganisationByOrgIdentifier(orgId);
         organisationCreationRequestValidator.isOrganisationActive(existingOrganisation);
 
-        OrganisationCreationRequestValidator.validateJurisdictions(newUserCreationRequest.getJurisdictions(), prdEnumService.getPrdEnumByEnumType(jurisdictionIds));
-
         List<PrdEnum> prdEnumList = prdEnumService.findAllPrdEnums();
         List<String> roles = newUserCreationRequest.getRoles();
-        UserCreationRequestValidator.validateRoles(roles, prdEnumList);
```

Comment:
same case here also. we should not delete the validation for roles while creating the organisation

Reply:
It's not been deleted, but the role validation method has been amended to reflect requirements that it only check there is at least one role.

Reply:
its fine